### DISCUSSION TITLE
Regarding GSoC 2024 application for the project Developing an Advanced Image Recognition Model for Bird Nest Detection in UAV Imagery.

### DISCUSSION THREAD

**Discussion by Aashish-100** [weecology/retriever#1679] [General]
Warm Greetings,
I'm Aashish Raheja, a Computer Engineering graduate from India. My major interests lie in using Data Science, Machine Learning, and Deep Learning to tackle climate change issues, particularly in geo-spatial data science for satellite image classification. I'm eager to contribute to the Advanced Image Recognition Model for Bird Nest Detection in UAV Imagery project and would value your insights on where to begin.
Thank you & Best Regards,
Aashish Raheja.

**Comment by henrykironde** [comment]
@Aashish-100 please ref to https://github.com/weecology/retriever/issues/1680
